setup.cfg

```
@@ -50,7 +50,7 @@ install_requires =
     docker
     importlib-metadata; python_version<"3.8"
     jsonschema>=4.4.0
-    protobuf~=3.20
+    protobuf==3.20.1
```

```suggestion
    protobuf~=3.20
```
Why not use compatible version like `Werkzeug`

---

what do you mean compatible version?
also that change is intended.